Source/DafnyTestGeneration/Main.cs

```diff
@@ -144,12 +143,8 @@ public static class Main {
           } catch (ArgumentException) {
             continue;
           }
-          var linePos = Utils.GetPosFromLine(uri, lineNumber, lineNumToPosCache);
-          var lineLength = Utils.GetPosFromLine(uri, lineNumber + 1, lineNumToPosCache) - linePos - 1;
-          var rangeToken = new RangeToken(new Token(lineNumber, 1), new Token(lineNumber, lineLength));
+          var rangeToken = new RangeToken(new Token(lineNumber, 1), new Token(lineNumber + 1, 0));
```

Comment:
Looking at the output, however, the highlighting is what I'd expect it to be, so I think it's fine to leave it like this.

Reply:
This is still inclusive, since columns are 1-based, so a 0th column does not exist. Essentially this is a trick that allows me to label the entire line without knowing the line's length.